content/docs/intro/console/collaboration/auditing.md

```diff
@@ -28,19 +28,19 @@ of the call the user made.
 ## Viewing Audit Logs in the Console
 
 Audit logs are available to organizations with an Enterprise level subscription
-only. If you are an organization administrator, you can view your organization's audit logs,
-by selecting your organization from the organization drop down. Then click on the settings tab.
-On the left nav-bar you should see a tab called Audit Logs. Audit Logs allow you to view
+only. If you are an organization admin, you can view your organization's audit logs,
+by selecting your organization from the organization drop down. Then select the **Settings** tab.
```

Comment:
```suggestion
by selecting your organization from the organization drop down, then selecting the **Settings** tab.
```

"Choosing" the settings tab would be okay here too, just to avoid the "select" repetition.